src/index.html

```diff
@@ -54,33 +54,33 @@
     <div class="menu-bar">
       <div class="container">
         <div class="row align-items-center">
-          <div class="col">
+          <div class="col search-wrapper">
             <form action="" class="search-form">
               <div class="category">
                 <i class="fas fa-list-ul"></i>
                 <div class="select-cta">
                   <p>Select a category</p>
                 </div>
                 <ul class="category-first">
-                  <li class="category-element">Option 1</li>
-                  <li class="category-element">
+                  <li>Option 1</li>
```

Comment:
dlaczego usuwasz tutaj klasy z elementów? Starajmy się robiąc coś nie zostawiać elementów HTML bez klas.

Reply:
Nieopatrznie przy rozwiązywaniu konfliktów, bardziej to przemyślę następnym razem. Dzięki za praktyczne uwagi.